src/DefaultManifold.jl

```diff
@@ -57,7 +57,7 @@ function check_approx(M::DefaultManifold, p, X, Y; kwargs...)
     return ApproximatelyError(v, s)
 end
 
-distance(::DefaultManifold, p, q) = norm(p - q)
+distance(::DefaultManifold, p, q, r::Real = 2.0) = norm(p - q, r)
```

Comment:
```suggestion
distance(::DefaultManifold, p, q, r::Real = 2) = norm(p - q, r)
```
LinearAlgebra.jl has an integer default here so maybe that's a good idea.

Reply:
Sure.